### DISCUSSION TITLE
Drag-and-drop from Firefox disables Alt+Tab

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#126] [Ideas]
The drag and drop gesture from Firefox download list blocks Alt+Tab task switching. When dragging an item from the Firefox download list, Alt+Tab task switching does not work anymore.

On earlier versions of either Linux Mint Cinnamon or Firefox, this was not the case, and this is something I do frequently on Windows. I presume the problem is with Cinnamon, since dragging a file from the Firefox download list and switching tasks works on Windows.

I used to drag files from the Firefox download list into Nemo to copy or move them to a different directory, or into a terminal window to get the path or interact with the file through a command-line tool. Having this gesture blocked is highly inconvenient.

**Comment by HT-7** [comment]
It seems no one cares. For the years to come, this will quietly keep disturbing users' workflows. Closed.

**Comment by faizul726** [comment]
same here. looked about 10 different forums but no solution. reason why i keep switching between chromium and firefox

**Comment by ArgentVASIMR** [comment]
Happens to me too, but with opening links externally from CKAN (KSP mod loader) instead.
